include/enrico/comm.h

```diff
@@ -178,46 +205,110 @@ class Comm {
 };
 
 template<typename T>
-std::enable_if_t<std::is_scalar<std::decay_t<T>>::value> Comm::broadcast(T& value) const
+std::enable_if_t<std::is_scalar<std::decay_t<T>>::value>
+Comm::send_and_recv(T& value, int dest, int source) const
 {
-  this->Bcast(&value, 1, get_mpi_type<T>());
+  if (active() && dest != source) {
+    if (rank == source) {
+      MPI_Send(&value, 1, get_mpi_type<T>(), dest, 81620, comm);
```

Comment:
Huh, can't remember!  Many times, I'll use the current date, but I definitely didn't write this on 8/16/20.  I'm happy to change it to 1 or whatever, if it's a source of confusion (especially since the `MPI_Recv` uses `MPI_ANY_TAG`).